src/components/atoms/AdminButton/AdminButton.tsx

```diff
@@ -0,0 +1,27 @@
+import { ButtonHTMLAttributes, RefObject } from "react";
+import cn from "classnames";
+
+import * as TW from "./AdminButton.tailwind";
+
+export type ButtonVariant = "primary" | "secondary";
+
+interface AdminButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  variant?: ButtonVariant;
+  forwaredRef?: RefObject<HTMLButtonElement>;
+}
+
+export const AdminButton: React.FC<AdminButtonProps> = ({
+  children,
+  variant = "primary",
+  forwaredRef,
+  ...extraParams
+}) => (
+  <button
+    type="button"
+    className={cn("AdminButton", TW.general, TW[variant])}
+    ref={forwaredRef}
```

Comment:
[NOTE] There was a typo committed long time ago.

```suggestion
    ref={forwardRef}
```

Reply:
fixed: https://github.com/sparkletown/sparkle/pull/2812/commits/0f78efe3a381196c9e5b7cacf68aa5c14daa88bc